package.json

```diff
@@ -14,7 +14,8 @@
     "test:coverage": "jest . --coverage ",
     "release:check": "changeset status --verbose --since=origin/main",
     "release:publish": "yarn install && yarn build && changeset publish",
-    "release:version": "changeset version && yarn install --immutable"
+    "release:version": "changeset version && yarn install --immutable",
+    "watch": "packemon watch"
```

Comment:
We can run `yarn watch` along with `cd site && yarn dev` to get hot reload with doc site

Reply:
I understand the idea, but not a fan of just call it `watch` and being at the root. The question here should be, do we want to test the `src` or the `dist`?